From d35af95c5cba1b898a9b183076d1a88c534415b9 Mon Sep 17 00:00:00 2001
From: Michael Simacek <msimacek@redhat.com>
Date: Tue, 21 Oct 2014 10:23:08 +0200
Subject: [PATCH] [pom_editor] Fix missing space between xmlns declarations

---
 java-utils/pom_editor.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/java-utils/pom_editor.py b/java-utils/pom_editor.py
index ca8fe45..ce487e0 100644
--- a/java-utils/pom_editor.py
+++ b/java-utils/pom_editor.py
@@ -350,8 +350,8 @@ class Pom(XmlFile):
     NSMAP = {'pom': 'http://maven.apache.org/POM/4.0.0',
               'xsi': 'http://www.w3.org/2001/XMLSchema-instance'}
     NS = '{' + NSMAP['pom'] + '}'
-    XMLNS = ('xmlns="http://maven.apache.org/POM/4.0.0"'
-    'xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"'
+    XMLNS = ('xmlns="http://maven.apache.org/POM/4.0.0" '
+    'xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" '
     'xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 '
     'http://maven.apache.org/xsd/maven-4.0.0.xsd"')
     DEPENDENCY_NODE = 'pom:dependency'
-- 
1.9.3

