--- javapackages-4.6.0/python/javapackages/common/manifest.py.omv~	2017-05-12 21:47:51.351799735 +0200
+++ javapackages-4.6.0/python/javapackages/common/manifest.py	2017-05-12 21:48:22.460095008 +0200
@@ -60,7 +60,7 @@ class Manifest(object):
             try:
                 jarfile = ZipFile(self._path)
                 if "META-INF/MANIFEST.MF" in jarfile.namelist():
-                    mf = jarfile.open("META-INF/MANIFEST.MF", "rU")
+                    mf = jarfile.open("META-INF/MANIFEST.MF", "r")
             except IOError:
                 pass
         if mf is None:
--- javapackages-4.6.0/java-utils/pom_editor.py.omv~	2017-05-12 21:53:48.023200338 +0200
+++ javapackages-4.6.0/java-utils/pom_editor.py	2017-05-12 21:54:02.904342798 +0200
@@ -580,7 +580,7 @@ def pom_remove_parent(pom=None):
     """[POM location]"""
     try:
         pom.replace_xml(pom.xpath_query_element("/pom:project/pom:parent"),
-                        etree.Comment(" parent POM reference removed by maintainer --> "))
+                        etree.Comment(" parent POM reference removed by maintainer -> "))
     except PomQueryNoMatch:
         raise PomQueryNoMatch("POM doesn't specify parent.")
 
