--- javapackages-4.2.0/install.orig	2014-10-18 17:22:50.442097972 +0100
+++ javapackages-4.2.0/install	2014-10-18 17:23:41.828094187 +0100
@@ -125,8 +125,8 @@
 inst_data java-utils/maven_depmap.py "${javadir}-utils"
 inst_data java-utils/pom_editor.py "${javadir}-utils"
 
-inst_data target/macros.jpackage "${rpmconfigdir}/macros.d"
-inst_data target/macros.fjava "${rpmconfigdir}/macros.d"
+inst_data target/macros.jpackage "${sysconfdir}/rpm"
+inst_data target/macros.fjava "${sysconfdir}/rpm"
 
 inst_data target/build-classpath.1 "${mandir}/man1"
 inst_data target/build-jar-repository.1 "${mandir}/man1"
@@ -173,7 +173,7 @@
 inst_config etc/javapackages-config.json "${javaconfdir}"
 
 exec >files-local
-inst_data target/macros.xmvn "${rpmconfigdir}/macros.d"
+inst_data target/macros.xmvn "${sysconfdir}/rpm"
 
 inst_data java-utils/mvn_alias.py "${javadir}-utils"
 inst_data java-utils/mvn_artifact.py "${javadir}-utils"
