--- javapackages-4.6.0/install.0100~	2017-05-12 21:28:20.386262918 +0200
+++ javapackages-4.6.0/install	2017-05-12 21:29:07.980646313 +0200
@@ -127,8 +127,8 @@ inst_data java-utils/maven_depmap.py "${
 inst_data java-utils/pom_editor.py "${javadir}-utils"
 inst_data java-utils/request-artifact.py "${javadir}-utils"
 
-inst_data target/macros.jpackage "${rpmconfigdir}/macros.d"
-inst_data target/macros.fjava "${rpmconfigdir}/macros.d"
+inst_data target/macros.jpackage "${sysconfdir}/rpm"
+inst_data target/macros.fjava "${sysconfdir}/rpm"
 
 inst_data target/build-classpath.1 "${mandir}/man1"
 inst_data target/build-jar-repository.1 "${mandir}/man1"
