--- javapackages-3.4.2/bin/shade-jar.orig	2014-01-17 00:03:01.248973600 -0200
+++ javapackages-3.4.2/bin/shade-jar	2014-01-17 00:03:21.926975030 -0200
@@ -37,7 +37,7 @@ if [ $# -ne 4 ]; then
     exit 1
 fi
 
-if [ ! -x /bin/objectweb-asm3-processor ]; then
+if [ ! -x /usr/bin/objectweb-asm3-processor ]; then
     echo "$0: Unable to execute objectweb-asm3-processor." >&2
     echo "$0: Make sure that objectweb-asm3 is installed." >&2
     exit 1
